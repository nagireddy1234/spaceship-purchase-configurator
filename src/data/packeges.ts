export const basicPackage = ['Air conditioning', 'Cloth seats', 'Fm radio'];
export const sportPackage = ['Air conditioning', 'Cloth seats', 'Fm radio', 'Personal tech support'];

export const luxPackage = ['Air conditioning', 'Luxury seats', 'Fm radio', 'Chrome weels', 'Window tint', 'Subwoofer'];
