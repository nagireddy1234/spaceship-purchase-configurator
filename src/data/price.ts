export const colorPrice = [0, 100, 100];
export const powerPrice = [0, 150, 200];
export const warpPrice = [0, 1000];
export const optionPrice = [0, 100, 500];
