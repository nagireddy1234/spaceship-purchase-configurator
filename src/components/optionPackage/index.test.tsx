import React from 'react';
import renderer from 'react-test-renderer';
import { sportPackage } from '../../data/packeges';
import OptionPackage from './index';

it('Checking ColorSelector component rendring correctly or not.', () => {
    const home = renderer
        .create(<OptionPackage planName="Sport" price={100} data={sportPackage} selected={false} onClick={() => {}} />)
        .toJSON();
    expect(home).toMatchSnapshot();
});
