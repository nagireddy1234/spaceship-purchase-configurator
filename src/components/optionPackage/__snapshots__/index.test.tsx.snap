// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking ColorSelector component rendring correctly or not. 1`] = `
<div
  className="col-3 justify-content-start "
  onClick={[Function]}
>
  <p
    className="planName"
  >
    Sport
  </p>
  <p
    className="planPrice"
  >
    +
    100
    â‚¬
  </p>
  <ul
    className="unorderlist"
  >
    <li>
      <div
        className="arrowup "
      />
      Air conditioning
    </li>
    <li>
      <div
        className="arrowup "
      />
      Cloth seats
    </li>
    <li>
      <div
        className="arrowup "
      />
      Fm radio
    </li>
    <li>
      <div
        className="arrowup "
      />
      Personal tech support
    </li>
  </ul>
</div>
`;
